gettsim/data/e_st_abzuege.yaml

```diff
@@ -1,56 +1,70 @@
-kifreib:
+kifreib_s_exm:
   name:
-    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes sowie Betreuungs- und Erziehungs- oder Ausbildungsbedarf)
+    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes).
     en:
   description:
     de: § 32(6) EStG
     en:
   values:
     1984-01-01:
-      value: 221
-      note:
+      value: 110
+      note: Wert in Euro. Der Betrag laut Gesetz ist 216 DM.
     1986-01-01:
-      value: 1270
-      note:
+      value: 635
+      note: Art. 1 G. v. 26.06.1985 BGBl. I S. 1153. Wert in Euro. Der Betrag im Gesetz lautet 1242 DM.
     1990-01-01:
-      value: 1546
-      note: .
-    1994-01-01:
-      value: 2098
-      note:
+      value: 773
+      note: Wert in Euro. Wert laut Gesetz 1512 DM.
+    1992-01-01:
+      value: 1049
+      note: Wert in Euro. Wert laut Gesetz 2102 DM.
     1996-01-01:
-      value: 3203
-      note: .
+      value: 1601
+      note: Art. 1. G. v. 20.10.1995 BGBl. I S. 1250. Wert in Euro. Laut Gesetz 261 DM pro Monat.
     1997-01-01:
-      value: 3534
-      note: .
-    2000-01-01:
-      value: 5080
-      note: .
+      value: 1767
+      note: Wert in Euro. Laut Gesetz 288 DM pro Monat (ab 2000 3456 DM pro Jahr)
     2002-01-01:
-      value: 5808
-      note:
+      value: 1824
+      note: Art. 1 G. v. 19.12.2000 BGBl. I S. 1790
     2009-01-01:
-      value: 6024
-      note:
+      value: 1932
+      note: Art. 1 G. v. 22.12.2008 BGBl. I S. 2955
     2010-01-01:
-      value: 7008
-      note:
+      value: 2184
+      note: Art. 1 G. v. 22.12.2009 BGBl. I S. 3950
     2015-01-01:
-      value: 7152
-      note: geändert durch BGBl. 2015 Nr. 30
+      value: 2256
+      note: Art. 1 G. v. 16.07.2015 BGBl. I S. 1202
     2016-01-01:
-      value: 7248
-      note: geändert durch BGBl. 2015 Nr. 30
+      value: 2304
+      note: Art. 2 G. v. 16.07.2015 BGBl. I S. 1202
     2017-01-01:
-      value: 7356
-      note: geändert durch Artikel 8 v. 20.12.2016, BGBl I S. 3000
+      value: 2358
+      note: Art. 8 G. v. 20.12.2016 BGBl. I. S. 3000
     2018-01-01:
-      value: 7428
-      note: Art. 9 BGBL I. S. 3000 v. 20.12.2016
+      value: 2394
+      note: Art. 9 G. v. 20.12.2016 BGBl. I. S. 3000
     2019-01-01:
-      value: 7620
+      value: 2490
+      note: Art. 1 G. v. 29.11.2018 BGBl. I S. 2210
+kifreib_bea:
+  name:
+    de: Kinderfreibetrag. Betreuungs-, Erziehungs-, Ausbildungsfreibetrag.
+    en:
+  description:
+    de: §32 (6) EStG, seit 2000.
+    en:
+  values:
+    2000-01-01:
+      value: 774
+      note: Art. 1 G. v. 28.12.1999 BGBl. I. S. 2552. Wert in Euro. Wert laut Gesetz 1512 DM.
+    2002-01-01:
+      value: 1080
       note:
```

Comment:
please add the note key

Reply:
I've looked for it for quite some time. No idea what changed it. It was first changed to 774€, but in the Neufassung from 2002 it's changed to 1080, meaning that it applies to the whole fiscal year.